### DISCUSSION TITLE
Short-term roadmap

### DISCUSSION THREAD

**Discussion by SuperLlama88888** [SuperLlama88888/holoprint#18] [Announcements]
- [x] Multi-structure support
- [x] Customisable controls
- [x] Localisation/Translation
- [x] Auto-updating old structures
- [x] Retextured/Renamed control items
- [ ] Material list on website
- [x] Fixing small issues and adding small features (https://github.com/SuperLlama88888/holoprint/milestone/2)
- [ ] Better resource pack support
- [ ] "Performance" mode with most features cut out, to reduce lag
- [ ] Windows and Android apps
- [ ] Support for including entities in holograms
- [ ] Behaviour pack for reliable player controls + block validation

**Comment by imv2r** [comment]
Go for it we all with u

Holoprint on top 🔝
